Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,15 +1,82 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+namespace BootCamp.Chapter1
 {
     public static class ArrayOperations
     {
+        // *************private helpers**********
         /// <summary>
+        /// Test that Array is neither null or empty.
+        /// </summary>
+        /// <returns>True if array can be manipulated by standard functions.</returns>
+        /// 
+        private static bool ValidateArrayForManipulation(int[] array)
```

Comment:
This function name sounds a bit artifically. I think a name like "IsNeitherNullNorEmpty" would fit better. Whereas a name like IsNullOrEmpty is preferable, but keep in mind to change your code properly then.